lib/components/app/app-menu.tsx

```diff
@@ -163,17 +184,55 @@ class AppMenu extends Component<
 
   render() {
     const {
+      activeLocale,
       callTakerEnabled,
+      configLanguages,
+      createOrUpdateUser,
       extraMenuItems,
       fieldTripEnabled,
       intl,
+      loggedInUser,
       mailablesEnabled,
       popupTarget,
       resetAndToggleCallHistory,
       resetAndToggleFieldTrips,
+      setLocale,
       toggleMailables
     } = this.props
 
+    const languageMenuItems: menuItem[] | undefined = configLanguages && [
+      {
+        children: Object.keys(configLanguages)
+          .filter((locale) => locale !== 'allLanguages')
+          .map((locale) => ({
+            iconType: <></>,
+            id: configLanguages[locale].name,
+            label:
+              activeLocale === locale ? (
+                <strong>{configLanguages[locale].name}</strong>
```

Comment:
An alternative to saying the current language in the heading would be to add invisible text saying that that is the selected language.

Reply:
This still needs to be resolved.

Reply:
Ok @miles-grant-ibigroup  and I agreed to solve the selected language indication in a separate PR.

Reply:
I'd like to use `aria-selected` if possible and that will likely require some syntax changes